l10n_it_withholding_tax/models/account.py

```diff
@@ -527,6 +527,22 @@ def _compute_payments_widget_reconciled_info(self):
                         payment_val["wt_move_line"] = False
         return
 
+    def action_register_payment(self):
+        """
+        Set net to pay how default amount to pay
+        """
+        res = super().action_register_payment()
+        amount_net_pay_residual = 0
+        for am in self:
+            if am.withholding_tax_amount:
+                amount_net_pay_residual += am.amount_net_pay_residual
+        if not float_is_zero(amount_net_pay_residual):
```

Comment:
if not float_is_zero(amount_net_pay_residual, precision_digits=2):

senza il secondo parametro va in errore

Reply:
Hai modo di testare l'attuale 14.0 facendo le stesse identiche operazioni? Non vedo come sulla 14 possa funzionare.

Reply:
Trovato prob. dovuto a #3322